lib/vmdb/plugins.rb

```diff
@@ -28,12 +28,20 @@ def init
       register_models
     end
 
-    def versions
+    def details
       each_with_object({}) do |engine, hash|
-        hash[engine] = version(engine)
+        hash[engine] = {
+          :name    => engine.name,
+          :version => version(engine),
+          :path    => engine.root.to_s
+        }
       end
     end
 
+    def versions
```

Comment:
who is the caller of this?

Reply:
The UI calls it to show the Plugins section of the About screen.  Ultimately this can go away in favor of calling details, but I didn't want to do a cross-repo change for this rake task.

There are tests on the versions method already that passed, so this still works.